native/src/main/java/io/ballerina/stdlib/grpc/callback/StreamingCallableUnitCallBack.java

```diff
@@ -118,6 +118,13 @@ public void notifySuccess(Object response) {
 
     @Override
     public void notifyFailure(io.ballerina.runtime.api.values.BError error) {
+        if (responseSender instanceof ServerCallHandler.ServerCallStreamObserver) {
+            ServerCallHandler.ServerCallStreamObserver serverCallStreamObserver = (ServerCallHandler
+                    .ServerCallStreamObserver) responseSender;
+            if (!serverCallStreamObserver.isReady() || serverCallStreamObserver.isCancelled()) {
```

Comment:
Shall we print a warning message and then ignore the error.

Reply:
Changed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/341/commits/ae8978336a5e1c7191915b7ff97c67efdae6437b